src/client.cpp

```diff
@@ -289,6 +289,9 @@ extern "C" {
 		pCapabilities->bSupportsRecordingPlayCount	= true;
 		pCapabilities->bSupportsLastPlayedPosition	= g_bEnableMultiResume;
 		pCapabilities->bSupportsRecordingEdl		= false;
+		pCapabilities->bSupportsRecordingsRename	= true;
```

Comment:
As long as the API function `RenameRecording` is implemented correctly this should be set to `true`, yes. I did no further investigations, but at least it currently does more than just `return PVR_ERROR_NOT_IMPEMENTED;` That's why I set this ti `true`.